src/devTools.js

```diff
@@ -247,11 +250,14 @@ export const ActionCreators = {
   setMonitorState(monitorState) {
     return { type: ActionTypes.SET_MONITOR_STATE, monitorState };
   },
-  recomputeStates(committedState, stagedActions) {
+  recomputeStates(committedState, stagedActions, timestamps, currentStateIndex=-1, skippedActions={}) {
```

Comment:
Can you describe the use case for this in more detail? How do you imagine this being used?